test/parallel/test-buffer-alloc.js

```diff
@@ -42,6 +42,7 @@ assert.strictEqual(d.length, 0);
 }
 
 // Test creating a Buffer from a Uint32Array
+// Note: it is implicitly interpreted as Array of integers modulo 256
```

Comment:
Honestly, that's not the behavior I'd have expected...

Reply:
`Buffer.from()` is not defined by documentation for any `ArrayBufferView` except for `Uint8Array` and `Buffer`. Even for `Uint8ClampedArray` which probably "works".

I agree that it's not what anyone would expect. There were related bugs in [node](https://github.com/nodejs/node/pull/43211) and [undici](https://github.com/nodejs/undici/pull/1472), and who knows how many similar bugs in ecosystem.

Not sure if this test was intentional or not, so probably it shouldn't be removed for now, just marked with a comment.
I'll add `uint8`-based test in case this is what was intended.